kernel/kernel-keymanager-service/Dockerfile

```diff
@@ -127,5 +127,5 @@ CMD if [ "$is_glowroot_env" = "present" ]; then \
     java -jar -javaagent:glowroot/glowroot.jar -Dloader.path="${loader_path_env}" -Dspring.cloud.config.label="${spring_config_label_env}" -Dspring.profiles.active="${active_profile_env}" -Dspring.cloud.config.uri="${spring_config_url_env}" -XX:HeapDumpPath=/home/  ./kernel-keymanager-service.jar ; \
     else \
     wget -q "${iam_adapter_url_env}" -O "${loader_path_env}"/kernel-auth-adapter.jar; \
-    java -jar -Dloader.path="${loader_path_env}" -Dspring.cloud.config.label="${spring_config_label_env}" -Dspring.profiles.active="${active_profile_env}" -Dspring.cloud.config.uri="${spring_config_url_env}" -Dspring.cloud.config.name="${spring_config_name_env}" ./kernel-keymanager-service.jar ; \
+    java -jar -Dloader.path="${loader_path_env}" -Dspring.cloud.config.label="${spring_config_label_env}" -Dspring.profiles.active="${active_profile_env}" -Dspring.cloud.config.uri="${spring_config_url_env}" ./kernel-keymanager-service.jar ; \

```

Comment:
why was spring cloud config name removed?

Reply:
Spring name was causing issue to connect to config server resulting pod to get to crasloop, we compared develop branch dockerfile with develop-java21 branch dockerfile we were not able to see the spring name in develop-java21 dockerfile so we removed the spring cloud config name, this resolved the issue and keymanager pod was up .